src/bitcoin-chainstate.cpp

```diff
@@ -85,7 +85,8 @@ int main(int argc, char* argv[])
         .datadir = gArgs.GetDataDirNet(),
         .adjusted_time_callback = NodeClock::now,
     };
-    ChainstateManager chainman{chainman_opts};
+    const node::BlockManager::Options blockman_opts{};
+    ChainstateManager chainman{chainman_opts, blockman_opts};
```

Comment:
nit: Since `blockman_opts` is empty here, I think you can pass it directly instead of creating `blockman_opts` for it.
```cpp
ChainstateManager chainman{chainman_opts, {}};
```